src/components/BusinessCaseReview/AlternativeAnalysisReview/index.tsx

```diff
@@ -51,38 +52,34 @@ const AlternativeAnalysisReview = (values: AlternativeAnalysisReviewProps) => {
         className="bg-white easi-business-case__review-solutions-wrapper"
         style={{ overflow: 'auto' }}
       >
-        {(tab => {
-          switch (tab) {
-            case '"As is" solution':
-              return <AsIsSolutionReview solution={asIsSolution} />;
-            case 'Preferred solution':
-              return (
-                <ProposedBusinessCaseSolutionReview
-                  name="Preferred solution"
-                  solution={preferredSolution}
-                />
-              );
-            case 'Alternative A':
-              return (
-                <ProposedBusinessCaseSolutionReview
-                  name="Alternative A"
-                  solution={alternativeA}
-                />
-              );
-            case 'Alternative B':
-              if (alternativeB && hasAlternativeB(alternativeB)) {
-                return (
-                  <ProposedBusinessCaseSolutionReview
-                    name="Alternative B"
-                    solution={alternativeB}
-                  />
-                );
-              }
-              return null;
-            default:
-              return <div />;
-          }
-        })(activeSolutionTab)}
+        <PrintableTabContent visible={activeSolutionTab === '"As is" solution'}>
```

Comment:
This is the most invasive change that I had to make. I don't think that rendering all tab content and hiding it with CSS (which is what we're now doing) is a serious performance issue based on what we're actually displaying.